build.Dockerfile

```diff
@@ -31,30 +31,13 @@ ENV SGX_MODE SW
 ARG WORKER_MODE_ARG
 ENV WORKER_MODE=$WORKER_MODE_ARG
 
-COPY . /root/work/worker/
 WORKDIR /root/work/worker
+COPY . .
 
-#RUN --mount=type=cache,target=/usr/local/cargo/registry \
-#	--mount=type=cache,target=/root/work/worker/target \
-#	make
 RUN make
 
-### Enclave Test Stage
-##################################################
-FROM builder AS enclave-test
-
-WORKDIR /root/work/worker/bin
-
-CMD ./integritee-service test --all
-
-
-### Cargo Test Stage
-##################################################
-FROM builder AS cargo-test
-
-WORKDIR /root/work/worker
+RUN cargo test --release
```

Comment:
`cargo test` is now run as part of the `docker build` process, instead of running `cargo test` on an image. For this change here it doesn't really matter - but it was necessary for my next evolution with docker layer caching.

Reply:
Nice, sounds promising. :)

Reply:
That's a good point - when I first started working with our setup, that was my thought too. 
Unfortunately, there is a difference between `cargo clippy` and `cargo test`. In an ideal world, `cargo test` would run completely in parallel, like `cargo clippy`. However, `cargo test`, for some reason, needs the enclave and sgx binaries. That means, you cannot run `cargo test` without having run `make` beforehand (which builds those binaries)

Reply:
Hm. Interesting. Never noticed before. Thanks for elaborating!